src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java

```diff
@@ -43,10 +43,18 @@ public ApiResponse<ChargesApi> getCharges(String contextId, String companyNumber
 
         Map<String, Object> logMap = createLogMap(companyNumber, "GET", uri);
         logger.infoContext(contextId, String.format("GET %s", uri), logMap);
-        return executeOp(contextId, "getCharges", uri,
-            getApiClient(contextId)
-                .privateDeltaChargeResourceHandler()
-                .getCharges(uri));
+        ApiResponse<ChargesApi> result = null;
+        try {
+            result = executeOp(contextId, "getCharges", uri,
+                getApiClient(contextId)
+                    .privateDeltaChargeResourceHandler()
+                    .getCharges(uri));
+        } catch (Exception ex) {
+            String message = ex.getMessage();
+            logger.info(message);
```

Comment:
Exception logged as info, do we need to this new try/catch?

Reply:
Was me trying to find out why integration test was behaving unexpectedly. Should have been removed, blame being tired on forgetting that!